Jenkinsfile

```diff
@@ -1,8 +1,65 @@
 #!/usr/bin/env groovy
 
-common {
-    slackChannel = '#tools-notifications'
-    downStreamValidate = false
-    runMergeCheck = false
-    nodeLabel = 'docker-oraclejdk8'
-}
+// Copyright 2020 Confluent Inc
```

Comment:
can we still use runJob from jenkins-common for this?

Reply:
I use an extra parameter to control whether we want to release to maven-central, I do not think jenkins-common currently have similar method, but I can add a new method but I am not sure if we want to do this since it is currently a single use case.

Reply:
I'm not suggesting you create a new groovy script in jenkins-common. You can use runJob here and define additional build parameters.